modules/ivc_champva/lib/pega_api/configuration.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/raise_custom_error'
+
+module IvcChampva
+  module PegaApi
+    class Configuration < Common::Client::Configuration::REST
+      def base_path
+        'https://bt41mfpkj5.execute-api.us-gov-west-1.amazonaws.com/prod/'
+      end
```

Comment:
Not a blocker, but we may want to move this into an AWS parameter at some point in the future.

Reply:
I was thinking about that, but this is how paths are being stored elsewhere in [similar code](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/dhp_connected_devices/lib/fitbit/configuration.rb) so I decided to err on the side of consistency.